eng/pipelines/client.yml

```diff
@@ -89,29 +89,16 @@ jobs:
           goals: 'install'
 
       - task: Maven@3
-        displayName: 'Install all client libraries (for SpotBugs analysis)'
+        displayName: 'Install Module and Run SpotBugs, Checkstyle, and Javadoc'
```

Comment:
Assume that ```eng/pipelines/client.yml``` doesn't exist (I'm actively trying to delete it :P). That means that all analysis type processes must be scoped to the libraries being build by a particular pipeline. Ideally they plug into the usual maven lifecycle.

Reply:
Is this the change for using `-pl` and listing out the projects to install and verify instead of using `pom.service.xml`? If that is the case this should continue to function the same as all the modules listed in `pom.service.xml` are leveraging their parent POMs build and validation configurations.